packaging/rpm/microshift.spec

```diff
@@ -165,6 +165,7 @@ install -d -m755 %{buildroot}/%{_unitdir}
 install -p -m644 packaging/systemd/microshift.service %{buildroot}%{_unitdir}/microshift.service
 
 install -d -m755 %{buildroot}/%{_sysconfdir}/microshift
+install -d -m755 %{buildroot}/%{_sysconfdir}/microshift/manifests
```

Comment:
Do we need /usr/lib/microshift and /usr/lib/microshift/manifests, too? Maybe we should add those in another PR, in case we change the names?

Reply:
The /usr/lib/microshift directory is not created at present.
Since we're talking about read-only directory structure that can only be updated from RPM, I thought it does not make sense to create an empty hierarchy.

Reply:
OK, I thought we might need to declare that we own that directory so that other RPMs can just put files into it.